### ISSUE TITLE
Error while running through CLI

### ISSUE THREAD

**Issue by avr6ude** [bamlab/flashlight#148]
Command I'm running: 
```bash
flashlight test --bundleId "my.app" \
  --testCommand "adb shell monkey -p my.app -c android.intent.category.LAUNCHER 1" \
  --duration 10000 \
  --resultsFilePath results.json
```
Error I'm getting: 
```bash
[18:06:53] 🚨 Error when running "adb shell monkey -p my.app -c android.intent.category.LAUNCHER 1":
script: tcgetattr/ioctl: Operation not supported on socket
```

Tried running it with `sudo`, both on physical device and an emulator. Tried running through `maestro` as well. Still getting the same error. 
For some reason, `flashlight measure` seems to work without issues so does the `adb` command alone. I can attach to the app and measure perfomance. So this issue seems to be CLI only.

**Comment by niebo26** [comment]
Hello!
Thank you for using Flashlight and writing this issue!
Would it be possible to have a bit more information on what you did and especially what you used to run the command please?
OS from where you launched the command and the devices you ran it onto (physical and emulator) most notably
Thank you!

**Comment by FilipMaslovaric** [comment]
Getting the exact same error when trying to run `maestro` through flashlight.

Operating System: MacOS Sonoma 14
Test Platform: Android
Test Device: Physical, USB Debugging, Android 13
IDE: Android Studio Giraffe | 2022.3.1 (Managing AVD), VSCode 1.82.2 (Running commands, code editing)
Java Runtime version: 17.0.6+0-17.0.6b829.9-10027231 aarch64
React Native version: 0.71.13
Flashlight version: 0.10.1
Maestro version: 1.32.0

(might be a good idea to have a bug submission form...)

**Comment by FilipMaslovaric** [comment]
> Thanks @FilipMaslovaric for the details! 🙏 
> 
> I'm able to reproduce this, I'll fix asap 🤞 

Thanks for the quick fix! Will try it out soon and report back if there are outstanding issues but I trust it's working now 🙇🏻‍♂️

**Comment by Almouro** [comment]
Should be fixed in version `0.11.1` of the CLI! 🥳 
To update: `curl https://get.flashlight.dev | bash`
